client/src/components/Search.js

```diff
@@ -57,16 +57,16 @@ const Search = (props) => {
     setState({ ...state, [e.target.name]: e.target.value });
   };
   const handleSubmit = async () => {
```

Comment:
I just pushed an axios file for FE to call Api easier. would you have a look and apply it here. it's like:
 const handleSubmit=async ()={
const response= await skyscannerApi.getQuotes(data);
...
}
It's just my opinion, what do you think?

Reply:
I took a look and looks good! I'll need to wait until your files are merged as there are likely edits I need to make to the front end components to accommodate. Once you merge I will update from the dev and dig into it!